### PR TITLE
Account Signing/NTP: More improvements and fixes

### PR DESCRIPTION
**Pull request by ripcurlx**
- Added capability for Refund Agent to make it backwards compatible
- Removed old DAO popup and added a new one for account signing and ntp
- other minor improvements (see commits)

### CODE REVIEW

File: desktop/src/main/java/bisq/desktop/main/overlays/windows/NewTradeProtocolLaunchWindow.java [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+    private void createContent() {
+        HBox content = new HBox();
+        content.setMinWidth(700);
+        content.setAlignment(Pos.CENTER);
+        content.setSpacing(20);
+
+        VBox accountSigning = getFeatureBox(Res.get("popup.news.launch.accountSigning.headline"),
+                Res.get("popup.news.launch.accountSigning.description"),
+                "image-account-signing-screenshot",
+                "https://docs.bisq.network/payment-methods#account-signing");
```
Comment (sqrrm) [inline desktop/src/main/java/bisq/desktop/main/overlays/windows/NewTradeProtocolLaunchWindow.java:126]: This link is broken

**Review by sqrrm — CHANGES_REQUESTED** [review]
Added a PR to fix the account signing changes

Broken link

File: desktop/src/main/java/bisq/desktop/main/overlays/windows/NewTradeProtocolLaunchWindow.java (hunk 1 above)
Comment (ripcurlx) [reply desktop/src/main/java/bisq/desktop/main/overlays/windows/NewTradeProtocolLaunchWindow.java:126]: @m52go This is a page that will be added, right?

**Review by sqrrm — APPROVED** [review]
ACK

**Pull request merged**